test/request_headers.spec.js

```diff
@@ -0,0 +1,153 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import test from 'ava'
+
+import Evaporate from '../evaporate'
+
+import initResponse from './fixtures/init-response'
+import completeResponse from './fixtures/complete-response'
+
+
+// constants
+
+const CONTENT_TYPE_XML = { 'Content-Type': 'text/xml' }
+const CONTENT_TYPE_TEXT = { 'Content-Type': 'text/plain' }
+
+const AWS_BUCKET = 'bucket'
+const AWS_UPLOAD_KEY = 'tests'
+
+const baseConfig = {
+  signerUrl: 'http://what.ever/sign',
+  aws_key: 'testkey',
+  bucket: AWS_BUCKET,
+  logging: false
+}
+
+const baseAddConfig = {
+  name: AWS_UPLOAD_KEY,
+  file: new File({
+    path: '/tmp/file',
+    size: 50
+  })
+}
+
+let server,
+    requestHeaders = {}
```

Comment:
I'd keep the declaration global and initialize it in `test.beforeEach()` which is run before each test.

```js
test.before(() => {
  requestHeaders = {}
})
```